### PR TITLE
AWS Explorer error node now provides a little more detail from AwsServiceExceptions

### PR DESCRIPTION
**Pull request by awschristou**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

Show a little more detail in the AWS Explorer Node when resources cannot be loaded

## Related Issue(s)

#634 #578 

## Screenshots (if appropriate)
image

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] Local run of `gradlew check` succeeds
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNode.kt [hunk 1]
```diff
@@ -193,22 +194,34 @@ class AwsExplorerLoadingNode(project: Project) :
...
     override fun update(presentation: PresentationData) {
         presentation.apply {
             // If we don't have a message, at least give them the error type
             tooltip = value.message ?: value.javaClass.simpleName
-            addText(MSG, SimpleTextAttributes.ERROR_ATTRIBUTES)
+
+            val exception = value
+            val errorDetails = if (exception is AwsServiceException) {
+                val awsErrorDetails = exception.awsErrorDetails()
+                "${awsErrorDetails.serviceName()} | ${awsErrorDetails.errorCode()}"
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNode.kt]: Not sure how I feel about ` | `, what about `: `. No space before, only after

Comment (awschristou) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNode.kt]: Coming up
image

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNode.kt]: Me likely.

**Comment by abrooksv** [comment]
We need to start change log entries again.

**Pull request merged**